### DISCUSSION TITLE
highlight_color - different colors for active and inactive groups

### DISCUSSION THREAD

**Discussion by elkrien** [qtile/qtile#2777] [Ideas]
Hi, 
Any chance to add possibility to have different colors of line / border for active and inactive groups?
For example my groups have white color when are inactive (no window / application is open in this group) and would like to have white line as a highlight when I'm at this group. But when group is busy it's name is red and I would like to have red line as a highlight whan i'm at this group. Also when I'm at empty (inactive) group my highlight and group name is white and when I open any window line and text color changes color to red - now only text changes it's color and line is still white.
So for example instead one parameter now `highlight_color` we have two: `highlight_color_active` and `highlight_color_inactive`.

**Comment by m-col** [comment]
I think it might be a relatively simple addition, though it would be preferable to not change the name of `highlight_color` for active windows as to not break user configs that use it. PR welcome!
